README.md

```diff
@@ -1823,7 +1823,7 @@ Sets the list of resources to look for when a client requests an index of the di
 
 ##### `docroot`
 
-**Required**. Sets the [`DocumentRoot`][] location, from which Apache serves files.
```

Comment:
I was explicitly passing undef for $docroot. Maybe keep it commented as a required param, but document that it can be explicitly undef.